soap/src/java/com/zimbra/soap/account/message/GetAddressListMemberResponse.java

```diff
@@ -0,0 +1,108 @@
+package com.zimbra.soap.account.message;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.google.common.collect.Lists;
+import com.zimbra.common.soap.AccountConstants;
+import com.zimbra.soap.type.ZmBoolean;
+
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2018 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+
+@XmlAccessorType(XmlAccessType.NONE)
+@XmlRootElement(name = AccountConstants.E_GET_ADDRESS_LIST_MEMBERS_RESPONSE)
+public class GetAddressListMemberResponse {
+
+    /**
+     * @zm-api-field-tag more-flag
+     * @zm-api-field-description 1 (true) if more members left to return
+     */
+    @XmlAttribute(name = AccountConstants.A_MORE /* more */, required = false)
+    private ZmBoolean more;
+
+    /**
+     * @zm-api-field-tag total
+     * @zm-api-field-description total number of distribution lists (not
+     *                           affected by limit/offset)
+     */
+    @XmlAttribute(name = AccountConstants.A_TOTAL /* total */, required = false)
```

Comment:
The total seems to always be returned in the implementation, is it not required?

Reply:
Fixed